bip-tombriar-compressed-transactions.mediawiki

```diff
@@ -0,0 +1,301 @@
+<pre>
+  BIP: ?
+  Layer: API/RPC
+  Title: Compressed Transactions RPC
```

Comment:
This document seems to be focused on describing the scheme for compressing transactions and only mentions the RPC in context of the reference implementation. Perhaps it would make sense to drop the "RPC" from the title.
```suggestion
  Title: Compressed Transactions
```

Reply:
Fixed, Thanks!